k8s/namespaces/jenkins/patches/ptlsbox/cluster-00/jenkins.yaml

```
@@ -263,6 +263,17 @@ spec:
             jenkins:
               clouds:
                 - azureVM:
+                    cloudTags:
+                      - name: "application"
+                        value: "core"
+                      - name: "builtFrom"
+                        value: "hmcts/shared-services-flux"
+                      - name: "businessArea"
+                        value: "Cross-Cutting"
+                      - name: "criticality"
+                        value: "High"
+                      - name: "environment"
+                        value: "ptlsbox"
```

https://github.com/hmcts/terraform-module-common-tags/blob/master/main.tf#L44
```suggestion
                        value: "sandbox"
```